### ISSUE TITLE
AU Linux kernel 2016 запрос на проверку Задание 6 Ялышева Наталья

### ISSUE THREAD

**Issue by rbbelka** [rbbelka/au-linux-kernel-spring-2016#7]
https://github.com/rbbelka/au-linux-kernel-spring-2016/commit/aa26beff22cea39a973600eba3e0e32e123bb310

**Comment by eabatalov** [comment]
Hello. I should be blind not to see the origins of your solution. This is not bad if you've studied this code and understood how it works.
We'll discuss it on the next LK class and you'll be able to get your LK credit.
I can't give you any points for this task because you'll get LK credit automatically if I give it to you. And this is not fare and right.

**Comment by rbbelka** [comment]
@eabatalov 
Thanks for reply.
LK course is additional elective course for me, third after two required courses. Under lack of time (because of being abroad) for this task I set myself goal just to understand it. It can be seen by comments in code which I added while studying this code.
It will be good for me to discuss it on purpose of better understanding in a first place and just then because of credit.
